core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileClassWarehouse.scala

```diff
@@ -484,17 +484,26 @@ case class FailedStagesProfileResults(appIndex: Int, stageId: Int, stageAttemptI
   }
 }
 
-case class FailedJobsProfileResults(appIndex: Int, jobId: Int,
-    jobResult: String, endReason: String) extends ProfileResult {
-  override val outputHeaders = Seq("appIndex", "jobID", "jobResult", "failureReason")
+case class FailedJobsProfileResults(
+    appIndex: Int,
+    jobId: Int,
+    sqlID: Option[Long],  // sqlID is optional because Jobs might not have a SQL (i.e., RDDs)
+    jobResult: String,
+    endReason: String) extends ProfileResult {
+  override val outputHeaders = Seq("appIndex", "jobID", "sqlID", "jobResult", "failureReason")
```

Comment:
Added the `sqlID` column. The rest of changes are code formatting to have each field in its own line.